doc/modules/svm.rst

```diff
@@ -599,57 +632,68 @@ Its dual is
    \textrm {subject to } & y^T \alpha = 0\\
    & 0 \leq \alpha_i \leq C, i=1, ..., n
 
-where :math:`e` is the vector of all ones, :math:`C > 0` is the upper bound,
-:math:`Q` is an :math:`n` by :math:`n` positive semidefinite matrix,
+where :math:`e` is the vector of all ones,
```

Comment:
maybe leave C in and say it's the upper bound for alphas?

Reply:
ok done